setup.py

```diff
@@ -26,13 +28,55 @@
 LONG_DESCRIPTION = readme
 AUTHOR = "Facebook AI Research"
 LICENSE = license
-REQUIREMENTS = (reqs.strip().split("\n"),)
+REQUIREMENTS = reqs.strip().split("\n")
+DEFAULT_EXCLUSION = ["test", "examples"]
+
+FULL_REQUIREMENTS = []
+# collect requirements.txt file in all subdirectories
+for file_name in glob.glob("**/*requirements.txt", recursive=True):
```

Comment:
What did we finalize on moving this out of the package as a separate package?

Reply:
Standard file name is `requirements.txt`. Why are you looking for `*requirements.txt`. It would be good to follow standard for this.